src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateKeyspaceIntegrationTest.java

```diff
@@ -159,12 +159,12 @@ public void invalidCommand() {
           .then()
           .statusCode(200)
           .body("$", responseIsError())
-          .body("errors[0].errorCode", is("COMMAND_FIELD_INVALID"))
-          .body("errors[0].exceptionClass", is("JsonApiException"))
+          .body("errors[0].errorCode", is(SchemaException.Code.UNSUPPORTED_SCHEMA_NAME.name()))
+          .body("errors[0].exceptionClass", is(SchemaException.class.getSimpleName()))
           .body(
               "errors[0].message",
               is(
-                  "Request invalid: field 'command.name' value `null` not valid. Problem: must not be null."));
+                  "The command attempted to create a keyspace with a name that is not supported.\n\nThe supported keyspace names must not be empty, more than 48 characters long, or contain non-alphanumeric-underscore characters.\nThe command used the unsupported keyspace name: null.\n\nResend the command using a supported keyspace name."));
```

Comment:
Could just check for part of message like

"The command attempted to create a keyspace with a name that is not supported."

so need not always change when we change error wording?